pkg/kubelet/kubelet.go

```diff
@@ -2961,6 +2968,10 @@ func (kl *Kubelet) handlePodResourcesResize(pod *v1.Pod, podStatus *kubecontaine
 			return nil, err
 		}
 		kl.statusManager.ClearPodResizePendingCondition(pod.UID)
+
+		// Clear any errors that may have been surfaced from a previous resize.
```

Comment:
nit: add a note that the condition will be added back as needed in the defer block, but this prevents errors from being preserved.

Reply:
done!